apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -166,6 +166,7 @@ const BookingPage = ({
           location,
           eventName: profile.eventName || "",
           bookingId: id,
+          isSuccessBookingPage: true,
```

Comment:
new parameter that defines if user is redirected to that page after successfully booking an event

Reply:
When does this variable turn false?

Reply:
always true when it comes from the booking page, but we also redirect to /success from bookingListItem to see the booking details. So it is here to check if we we redirected from the booking page or come from a bookingListItem